### PR TITLE
Optimize asset loading for docs

### PR DESCRIPTION
**Pull request by megawac**
for @aearly 

docs can be rendered without scripts as far as I can tell (especially without the bootstrap+typeahead scripts).

The downside of this is the javascripts (prettify and lang) seem to thrash the dom

### CODE REVIEW

**Comment by hargasinski** [comment]
Looks good to me. I'll have to move RunKit to the end of the body then too in #1317.

File: support/jsdoc/theme/tmpl/layout.tmpl [hunk 1]
```diff
@@ -85,8 +78,18 @@
     Documentation generated by <a href="https://github.com/jsdoc3/jsdoc">JSDoc <?js= env.version.number ?></a><?js if(env.conf.templates && env.conf.templates.default && env.conf.templates.default.includeDate !== false) { ?> on <?js= (new Date()) ?><?js } ?> using the Minami theme.
 </footer>
 
+
+<script src="scripts/prettify/prettify.js"></script>
+<script src="scripts/prettify/lang-css.js"></script>
```
Comment (megawac) [inline support/jsdoc/theme/tmpl/layout.tmpl]: @hargasinski do you know if these scripts have the `async` attribute or do they depend on one another?

File: support/jsdoc/theme/tmpl/layout.tmpl [hunk 2]
```diff
@@ -8,21 +8,14 @@
 
     <link rel="icon" href="favicon.ico?v=2" />
     <link src="text/css" rel="stylesheet" href="https://cdn.jsdelivr.net/bootstrap/3.3.6/css/bootstrap.min.css">
-    <script src="scripts/async.js"></script>
 
-    <script src="scripts/prettify/prettify.js"></script>
-    <script src="scripts/prettify/lang-css.js"></script>
```
Comment (megawac) [inline support/jsdoc/theme/tmpl/layout.tmpl]: turns out this one isn't necessary

**Comment by megawac** [comment]
Also, I uploaded a new version of prettify to jsdelivr (http://www.jsdelivr.com/projects/prettify) though none of those themes seem to work

**Pull request merged**